code/index.html

```diff
@@ -2,21 +2,83 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Newspaper</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    
   </head>
   <body>
+
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="Flode">
+        <div>🎢</div>
+        <div>🎢</div>
+        <div>🎢</div>
+      </div>
+
+      <nav>
+        <a href="https://images.unsplash.com/photo-1493780474015-ba834fd0ce2f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1026&q=80"
+        >Asia</a>
+        <a href="https://images.unsplash.com/photo-1491557345352-5929e343eb89?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80"
+        >Europe</a>
+        <a href="https://images.unsplash.com/photo-1547471080-7cc2caa01a7e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1051&q=80"
+        >Africa</a>
+        </nav>
+
+        <h3>Breaking news!</h3>
+
+        <img src="https://image.flaticon.com/icons/svg/2460/2460249.svg" alt="Hippo" id="hippo">
+
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <section class=" welcome-Bignews ">
+
+      <img src="https://images.unsplash.com/photo-1558285549-2a06f9a5fe65?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80" 
+      alt="Fruit img">
+
+      <div>
+        <h2>Things you no longer allowed to do! </h2>
+        <p>The climate change is here, and it is real. In order for us to live on our planet for a while we need to make some 
+          drastic changes. From now it is forbidden to fly. I know it sounds like a fake news, but it is´t. More and more things will be forbidden.
+          Below you can read about four new things that have been banned this week, in the whole world.</p>
+
+      </div>
 
-    <section class="">
-      <!-- The other news  -->
     </section>
+
+    <section class="product">
+      <div>
+      <img src="https://images.unsplash.com/photo-1504700610630-ac6aba3536d3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80"
+          alt="lake" class="product-img">
+      <h1>Be in the nature.</h1>
```

Comment:
Try using only one `<h1>` per page and instead use one of the other headings. The `<h1>` should describe what page it is